src/ggrc/assets/javascripts/controllers/tree_view_controller.js

```diff
@@ -1445,12 +1449,17 @@ can.Control("CMS.Controllers.TreeViewNode", {
   }
 
   , draw_node: function() {
+    if (this._draw_node_in_progress) {
```

Comment:
Instead of using a new variable for this, could we use the _draw_node_in_progress deferred instead?